### ISSUE TITLE
When using more than one AUv3 instance SoundFonts fails to work

### ISSUE THREAD

**Issue by R-a-l-f** [bradhowes/SoundFonts#5]
Hi,

using a single instance of SoundFonts as AUv3 plugin works, two or more instances fail to work, if the host is Cubasis 3. There is no issue with SoundFonts, if the host is MultitrackStudio.

iPad Pro (12.9-inch) (3rd generation) MTFR2FD/A
iPadOS 15.5 (19F77)
SoundFonts Version 2.24.
Cubasis 3 Version 3.4.3
MultitrackStudio Version 4.5.

Cubasis support ask to report against SoundFonts and wrote: "[snip] If required, we’re at had to have them equipped with a Cubasis 3 promo code for testing purposes. [snip]"

Regards,
Ralf

**Comment by bradhowes** [comment]
Hi Ralf. Thanks for bringing up this issue. I had a problem in the past with Cubasis. This could be a regression or another issue I introduced with more recent changes. I will take a look to see if I can reproduce your issue.

**Comment by bradhowes** [comment]
Hmmm. I did not realize I already had Cubasis on my new iPad so I started it up. Still had my old test song on it with two SoundFonts tracks. I added a third just for fun.

![image](https://user-images.githubusercontent.com/686946/178538834-881088e5-710a-484c-ae29-e0fa084e5222.jpeg)

I hear all three tracks and can solo each one individually.

What exactly happens in your situation? Does one or more of the plugins crash?

**Comment by R-a-l-f** [comment]
Hi,

I've got a 55 seconds short soundtrack at hand made with Cubasis 3 a few days ago. Among other audio units it includes 3 instances of SoundFonts and the issue happened very often after editing a MIDI track. I duplicated the soundtrack, started a screen recording to demonstrate the issue, but I couldn't reproduce it myself. I tried for 3 minutes, but everything was stable this time.

When the issue appeared a few days ago, the SoundFonts tracks suddenly stopped making sound. I needed to restart Cubasis and after that the SoundFonts tracks were ok for a while, often just for a few seconds.
I've got no idea why it's stable right now.

However, I will do the next project with MultitrackStudio.

Regards,
Ralf
